src/sources/dnstap/mod.rs

```diff
@@ -195,8 +203,20 @@ impl FrameHandler for DnstapFrameHandler {
 
         let mut log_event = LogEvent::default();
 
+        self.log_namespace.insert_vector_metadata(
+            &mut log_event,
+            self.timestamp_key().as_str(),
+            "ingest_timestamp",
```

Comment:
@fuchsnj did we want to go ahead and `path!()` all of these while we're making these changes? Ex: https://github.com/vectordotdev/vector/pull/14985/files#diff-ce8ca490d9ddbe754a3227df21f669c2f7c95198ca26c9067076ad364617bbcdR559-R560